src/app/hm-admin/+list/list.component.ts

```diff
@@ -22,7 +24,13 @@ export class ListComponent implements OnActivate {
 
   properties: Property[] = [];
 
-  collection: any;
+  collection: any = {
+    currentPage: 0,
+    totalItems: 0,
+    itemsPerPage: 0
+  };
+
+  itemsPerPageOptions: number[] = [10, 30, 50];
```

Comment:
It do it [right here](https://github.com/Gregcop1/hm-admin/blob/c9b9b65fd0e97305abb33f2cd0ab1ab0567ba8b3/src/app/hm-admin/%2Blist/list.component.ts#L56). But I need a default configuration if the user don't fill it.